.github/workflows/ci.yaml

```diff
@@ -2,6 +2,8 @@ name: CI
 
 on:
   push:
+    branches:
```

Comment:
I'd keep the integration tests on pushes, I think they can be valuable before creating the PR. However I'd remove the integration tests on PR.

Reply:
Why do we want to specifically trigger the CI on every push just in this repository?

Reply:
We do trigger CI's on every push in most repos, don't we?

Reply:
Fixed.